src/locales/en.json

```
@@ -1,4 +1,43 @@
+  "about_nft_page_nft_portfolio_title": "Collect as you read",
+  "about_nft_page_nft_portfolio_description": "<p>A beautiful <b class=\"text-like-green\">NFT collection display</b>. Collect to showcase an individual’s <b class=\"text-like-green\">readership</b> and <b class=\"text-like-green\">authorship</b> with a verifiable ownership record. Show off your collection!</p><p class=\"mt-[16px]\">Writers can <b class=\"text-like-green\">build communities</b> with like-minded readers, and use NFT to provide unique experiences on readers' interactions and participation.</p>",
+  "about_nft_page_nft_portal_title": "Publish any website as NFT",
+  "about_nft_page_nft_portal_description": "<p>Writers can turn creative works and <b class=\"text-like-green\">URLs</b> into NFTs.</p><p class=\"mt-[16px]\">Enjoy the benefits of decentralized storage and create NFT in several clicks. No more pages not found, always <b class=\"text-like-green\">accessible</b> to your content, truly <b class=\"text-like-green\">own</b> and <b class=\"text-like-green\">monetize</b> your work.</p>",
+  "about_nft_page_dynamic_pricing_title": "Get rid of sales hurdle",
+  "about_nft_page_dynamic_pricing_description": "<p>Write as usual; publish differently.</p><p class=\"mt-[16px]\">The preset <b class=\"text-like-green\">bonding curve</b> design sets all Writing NFT prices with <b class=\"text-like-green\">dynamic pricing</b>. Each Writing NFT will automatically increase in price after each NFT is collected. <b class=\"text-like-green\">Limitless supply</b>, writers do not have to worry about maintaining floor price or supply quantity.</p>",
+  "about_nft_page_faq_title": "FAQ",
+  "about_nft_page_join_the_community_title": "加入社群",
+  "faq_list": [
+    {
+      "q": "Why publish articles as Writing NFT?",
+      "a": "An individual’s content portfolio can be viewed as part of a reliable decentralized identity in Web3. It’s a record of individual authorship. Writing NFT can guarantee the immutability of content and later verifiable with proof of ownership and property rights."
+    },
+    {
+
+      "q": "How to get started with Writing NFT?",
+      "a": "There are three ways, 1) if you are a WordPress user, you can directly install the free LikeCoin plugin from the WordPress plugins. 2) via NFT Portal, you can mint NFT from any URLs, and it will generate an NFT listing page for you. 3) via the partnered website. There are several media/ blogging platforms that have integrated with LikeCoin. {{list}}"
```

```suggestion
      "a": "There are three ways, 1) if you are a WordPress user, you can directly install the free LikeCoin plugin from the WordPress plugins. 2) via NFT Portal, you can mint NFT from any URLs, and it will generate an NFT listing page for you. 3) via the partnered website. There are several media/ blogging platforms that have integrated with LikeCoin."
```
Not sure why we need `{{list}}`. Will these platforms be listed in `{{list}}`?